tools/docker-compose.yml

```diff
@@ -3,14 +3,14 @@ services:
 
   boot:
     container_name: boot
-    command: boot tce run
+    command: boot node up --name test --home /tmp/node_config
     image: ghcr.io/topos-protocol/topos:main
     init: true
     labels:
       "autoheal": "true"
       "prometheus-job": "boot"
     healthcheck:
-      test: ./topos tce push-peer-list --node http://localhost:1340 --format json /tmp/shared/peer_ids.json && ./topos tce status --node http://localhost:1340
+      test: ./topos tce status --node http://localhost:1340
       interval: 15s
     volumes:
```

Comment:
You should add the key files here directly

Reply:
What exactly do you mean with key files?

Reply:
I mean all the files that you added in the `Dockerfile` should be part of the `docker-compose`